src/main/MainTabs.js

```diff
@@ -12,7 +12,7 @@ import SettingsCard from '../settings/SettingsCard';
 import { IconHome, IconStream, IconSettings } from '../common/Icons';
 import IconUnreadConversations from '../nav/IconUnreadConversations';
 
-export default TabNavigator(
+export default createMaterialTopTabNavigator(
```

Comment:
Wow, that is confusing that this works! I appreciate that the apparent contradiction is pointed out in the commit message. :-)

It doesn't actually seem to be documented that this is something you can do -- `tabBarPosition` doesn't appear on this page:
https://reactnavigation.org/docs/en/material-top-tab-navigator.html
As long as it does work, it's OK, but it's something we'd want to call out and be watchful for on future upgrades.

It looks like there is a "bottom" counterpart to this component, though:
https://reactnavigation.org/docs/en/material-bottom-tab-navigator.html
Does that one work? If so, that seems cleaner.

Reply:
... Aha -- I see #2831 switches to doing exactly this. And from looking at that commit, I see I'd missed this line in that doc:
> npm install react-navigation-material-bottom-tabs

I guess I'll continue this thread on that PR.